tests/Jellyfin.Server.Integration.Tests/Controllers/SessionControllerTests.cs

```diff
@@ -24,4 +24,18 @@ public async Task GetSessions_NonExistentUserId_NotFound()
         using var response = await client.GetAsync($"Session/Sessions?userId={Guid.NewGuid()}").ConfigureAwait(false);
         Assert.Equal(HttpStatusCode.NotFound, response.StatusCode);
     }
+
+    [Theory]
+    [InlineData("?nowPlaying=false")]
+    [InlineData("?nowPlaying=true")]
+    [InlineData("")]
+    public async Task GetSessions_NowPlaying_Ok(string querystring)
```

Comment:
This test is the same as testing for an invalid query parameter since the framework doesn't error on unknown query parameters. To actually test this you would need to set up the list of sessions to include playing & non playing ones.

Reply:
Yes. I wouldnt mind to, but i am unfamiliar with the codebase still, and dont know how to do that.

Shall i remove the testcase?

Reply:
The testcase itself is behaviour by the framework.

A test that you really get only active sessions when `nowPlaying=true` is in order